source/server/spongineer/logs.rst

```diff
@@ -9,6 +9,134 @@ logfiles from SpongeForge and SpongeVanilla servers including short descriptions
+a central log collection and analysis server. Read more about configuring appenders
+`here <https://logging.apache.org/log4j/2.x/manual/configuration.html#Appenders>`__.
+
+The ``Loggers`` section defines filters for loggers and which to which targets they should be forwarded to. This section
+is usually the section you have to edit if you want to mute a specific plugin in the logs. Let's look into this a bit
+more:
+
+.. code-block:: xml
+
+    <Logger level="info" name="com.example.mod"/>
+
+This will limit the logs of ``com.example.mod`` to ``info`` and higher messages. Beware this affects all output channels
+and also any logger that is created for a sub-package of the given path. Read more about filters
+`here <https://logging.apache.org/log4j/2.x/manual/configuration.html#Filters>`__.
+
+.. code-block:: xml
+
+    <Root level="all">
+        <AppenderRef ref="Console" level="info"/>
+        <AppenderRef ref="DebugFile"/>
+    </Root>
+    
+This section configures two output channels. First, the appender called ``Console`` with a log level filter of ``info``
+and higher, and last, the appender called ``DebugFile``. It is recommended to keep at least one logger to a persistent
+target such as a file for later error search.
+
+.. note::
+
+    If you are wondering, why your new plugin's log messages don't seem to show up: The Console's log level is
+    configured to be at least info by default which hides your debug messages.
+
+If you don't want to reconfigure the entire logging, but want to hide a certain plugin from the logs you can also use
+composite logging options. This can be achieved by referencing both the original logging config and you specialized
+config that does only contain the changed logging options. The following example shows this:
+
+.. code-block:: bash
+
+    java -Dlog4j.configurationFile=log4j2_server.xml,log4j2_custom.xml -jar server.jar
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <Configuration status="WARN">
+        <Loggers>
+            <Logger name="com.example.logspammer" level="off"/>
+        </Loggers>
+    </Configuration>
+
+In this example all logs from the ``com.example.logspammer`` package won't be shown or saved. For debugging purposes, it
+might be useful to include your plugin's log messages in the console so you don't have to tail the debug log. This can
+be achieved using the following example:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <Configuration status="WARN">
+        <Loggers>
+            <Logger name="com.example.newplugin" level="all" additivity="false">
+                <AppenderRef ref="Console"/>
+                [...]
+            </Logger>
+        </Loggers>
+    </Configuration>
+
+.. tip::
+
+    Logging affects performance. If you log too much you might lose a small amount of tps. If you don't have (complete)
```

Comment:
The third sentence here seems to be a non-sequitur. Perhaps "Also remember that ..." ?